projects/firefox/project.yaml

```diff
@@ -0,0 +1,6 @@
+homepage: "https://www.mozilla.org/firefox/"
+primary_contact: "agaynor@mozilla.com"
+fuzzing_engines:
+- none
+sanitizers:
+- address
```

Comment:
At a minimum we also want @choller in the CCs, probably some other folks as well